src/cancellable-fetch.ts

```diff
@@ -28,10 +28,15 @@ export const isFetchError = (error: unknown): error is FetchError =>
 const cancellableFetch = <T>(url: string, init?: RequestInitWithTimeout): CancellableFetch<T> => {
   const abortController = new AbortController();
   const abort = () => abortController.abort();
+  const bearerToken = process.env.REACT_BEARER_TOKEN;
 
   const fetchPromise = fetch(url, {
     ...init,
-    headers: { ...init?.headers, Accept: 'application/json' },
+    headers: {
```

Comment:
I guess we can remove these changes to `cancellableFetch` now that we're using `consoleFetchJSON` instead.